UI/Web/src/app/admin/manage-settings/manage-settings.component.html

```diff
@@ -33,6 +33,19 @@
             </div>
         </div>
 
+        <div class="mb-3">
+            <label for="settings-baseurl" class="form-label">Base Url (WARNING: Requires server restart)</label>&nbsp;<i class="fa fa-info-circle" placement="right" [ngbTooltip]="baseUrlTooltip" role="button" tabindex="0"></i>
+            <ng-template #baseUrlTooltip>Use this if you want to host Kavita on a base url ie) yourdomain.com/kavita (WARNING: Requires server restart)</ng-template>
+            <span class="visually-hidden" id="settings-cachedir-help">Use this if you want to host Kavita on a base url ie) yourdomain.com/kavita (WARNING: Requires server restart)</span>
+            <input id="settings-baseurl" aria-describedby="settings-baseurl-help" class="form-control" formControlName="baseUrl" type="text"
+            [class.is-invalid]="settingsForm.get('baseUrl')?.invalid && settingsForm.get('baseUrl')?.touched">
+            <div id="baseurl-validations" class="invalid-feedback" *ngIf="settingsForm.dirty || settingsForm.touched">
+              <div *ngIf="settingsForm.get('baseUrl')?.errors?.pattern">
+                  Base URL must start and end with /
```

Comment:
Per logic, it must start and end in a /, shouldn't it?

Reply:
I don't understand what you mean by this

Reply:
In Program.cs, you have code to ensure base url when fetched from appsettings.json starts and ends with /. In that case, might as well put that same logic in your inline validation.

Reply:
Ah, got it. It's already, in the validation, using a RegExp. See https://github.com/Kareadita/Kavita/pull/1824/files#diff-d63d0019b68d4b64af3d4280d3d19857e793b8cd7e1b13a29c864b98f9e178c0R50

Reply:
Correct, so don't you think the user should understand that as well? Hence why I want the message updated.

Reply:
Oh. I thought "Base URL must start and end with /" was already a good message. Suggestions to make it clearer?

Reply:
Okay my bad, I totally misread that as "Base URL must end with /". Nevermind, so we are good.